datastorage.lua

```diff
@@ -60,13 +60,20 @@ end
 
 local function initDataDir()
     local sub_data_dirs = {
-        "cache", "clipboard",
-        "data", "data/dict", "data/tessdata", "docsettings",
-        "history", "ota", "patches", "plugins",
-        "screenshots", "settings", "styletweaks",
+        "cache",
+        "clipboard",
+        "data",
```

Comment:
Good for one line per dir.
May be keep the ones a user may/will find, but we don't want created, commented out, as a reference of the koreader/ subdirectories ?
```lua
        "data",
        -- "docsettings", -- created when needed
        -- "history", -- legacy/obsolete sidecar files
        -- "patches", -- must be created manually by the interested user
```